src/Plugin.FilePicker/Plugin.FilePicker/Properties/AssemblyInfo.cs

```diff
@@ -26,5 +26,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.0.0")]
-[assembly: AssemblyFileVersion("1.3.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]

```

Comment:
All the version number changes are nice, but not necessary, this is typically done while building. No need to change it or anything, just wanted to let you know :)